RecoTracker/ConversionSeedGenerators/plugins/Conv4HitsReco.cc

```diff
@@ -1,11 +1,146 @@
-#define Conv4HitsReco_cxx
-#include "Conv4HitsReco.h"
-#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include <iostream>
 #include <sstream>
-// 1 -> 4
-// 2 -> 3
-// 3 -> 2
-// 4 -> 1
+#include <iomanip>
+#include <cmath>
+
+#include <TVector3.h>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+class Conv4HitsReco {
```

Comment:
this is not quite an appropriate change. This class is not a plugin. The declaration should be available in a header file.
It is kind of dead code though and is perhaps OK to be removed.

Reply:
yes looks like dead code, nothing is using it. +1 to drop it.